### ISSUE TITLE
Can this run in the browser? or made to?

### ISSUE THREAD

**Issue by faddat** [mafintosh/peerflix#299]
OK so I am a total noob to web front end.  Recently I have been playing with chocobozzz/peertube and I find the performance pretty disappointing.  I notice that you have a node.js stack for finding peers and for streaming torrents.  I'd love to run it in the browser, of course.  

Please, ELI5.

**Comment by asapach** [comment]
@faddat, check out [peerflix-server](https://github.com/asapach/peerflix-server).

**Comment by necronoise** [comment]
Yes you can, just put your ip with the port 8888 and done.
_Example: 192.168.0.1:8888_

Whatever computer or mobile in your wifi can also open the link in the browser.
